test/service/test_request_builder.py

```diff
@@ -5,15 +5,19 @@
 import logging.config
 import os
 import unittest
+from service import WebServiceTestCase
 from azul.service.responseobjects.elastic_request_builder import ElasticTransformDump as EsTd
-import azul.service.config
 from azul import config
 
-logging.config.fileConfig(os.path.join(os.path.dirname(azul.service.config.__file__), 'logging.conf'))
-logger = logging.getLogger("dashboardService")
+logger = logging.getLogger(__name__)
 
 
-class MyTestCase(unittest.TestCase):
+class TestRequestBuilder(WebServiceTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.data_directory = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
```

Comment:
Yeah. This looks better than f"" sttring method.

Reply:
OK. I will follow this format.